event_lead_event/wizard/lead_event.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class LeadEventConfirm(models.TransientModel):
+
+    _name = 'event.event.leadconfirm'
+
+    def _get_current_lead_event(self):
+        return self.env['event.event'].search([('lead_event', '=', True)])
+
+    def _get_event_id(self):
```

Comment:
Instead of defining this function, you can pass `'default_event_id': active_id` in the context of `action_event_lead_event`